Makefile.am

```diff
@@ -259,6 +258,16 @@ install-exec-hook:
 			rm -f $${e}.tmp ; \
 		fi ; \
 	done ; \
+	if test -e ${DESTDIR}${bindir}/@MPICC_NAME@ ; then \
+		if test "@MPICC_NAME@" != "@MPICC_ABI_NAME@" ; then \
+			cd ${DESTDIR}${bindir} && ln -f -s @MPICC_NAME@ @MPICC_ABI_NAME@ ; \
```

Comment:
This is the place I'm talking about. This symlink is created unconditionally, right?

Reply:
This is post-install scripts. It only runs for those scripts that are already installed. 
`if BUILD_ABI_LIB` condition should exclude those _abi scripts from being installed unless --enable-mpi-abi is set.

Reply:
OK. I'm confused, but that's simply because I don't fully understand how this business work. Let me try myself what happens with ABI turned off.

Reply:
@hzhou I was right, the `mpicc/cxx_abi` symlinks are created unconditionally, even when ABI is not enabled at configure time.

Reply:
Oh.. Now I think I know how to do it properly...